### DISCUSSION TITLE
Google and Apple 'Tax'

### DISCUSSION THREAD

**Discussion by cngeru** [google-pay/flutter-plugin#26] [General]
Want to use the flutter plugin in my apps. Will  the 15/30% fee apply ?

**Comment by jamesblasco** [comment]
Hello @cngeru. The `pay` plugin adds support for wallet payments, right now Google Pay, and Apple Pay in the future. The fee depends on the payment provider you use with it (See available options [here](https://developers.google.com/pay/api)).

Notice that, both Google Play and Apple App Store require you to use their In-App Purchases API for purchasing digital goods. And there is where the 15/30% fee applies. [in_app_purchase](https://pub.dev/packages/in_app_purchase) is the package that allows you to implement these payment processors.

For any other thing non-digital you can use this plugin to allow customers to pay with the wallet they have installed on their phones
